lib/conf_loader.rb

```diff
@@ -19,7 +19,7 @@ class ConfLoader
   def self.load(path, env)
     template = ERB.new File.new(path).read
     environments = YAML.load template.result(binding)
-    hash = environments[env]
+    hash = environments.fetch(env)
```

Comment:
Raising a more specific error subclass with more information (e.g. environment name & path where conf was looked for) would be useful for end user.